From b0ced6261ece39f9b6ac3cef07b06f02d3f1cbca Mon Sep 17 00:00:00 2001
From: Prasanth Ganapavarapu <prasanthkumar_ganapavarapu@epam.com>
Date: Thu, 3 Aug 2023 10:35:31 +0530
Subject: [PATCH] modify firstFile in git_1

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index 00935f1..e4aa73e 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -2,8 +2,8 @@ Line 1
 Line 2
 Line 3
 Line 4
-Line 5
-Line 6
+Line 5 -- modified
+Line 6 -- modified
 Line 7
 Line 8
 Line 9
-- 
2.39.0.windows.2

